types/node/index.d.ts

```diff
@@ -1782,6 +1782,9 @@ declare module "child_process" {
         stdio: [stream.Writable, stream.Readable, stream.Readable];
         pid: number;
         kill(signal?: string): void;
+        send(message: any): boolean;
```

Comment:
I think this one is not needed as the next one with optional callback should cover the `send(message)` and `send(message, cb)` use cases.